typescript/ts-node-lowering/src/org/jetrbains/dukat/nodeLowering/lowerings/introduceModels.kt

```diff
@@ -351,6 +351,9 @@ internal class DocumentConverter(private val moduleNode: ModuleNode, private val
                     static = static,
                     override = null,
                     immutable = getter && !setter,
+                    initializer = with(ExpressionConverter(this@DocumentConverter)) {
```

Comment:
do we really need with here?

Reply:
What's the alternative?

Reply:
We need to call an extension function defined in other class, I don't know other way to do this